src/Nri/Ui/TextArea/V3.elm

```diff
@@ -88,61 +88,38 @@ contentCreation model =
 view_ : Theme -> Model msg -> Html msg
 view_ theme model =
     let
-        minHeight =
+        autoresizeAttrs =
             case model.height of
+                AutoResize _ ->
+                    [ Attributes.attribute "data-autoresize" "" ]
+
                 Fixed ->
                     []
-
-                AutoResize minimumHeight ->
-                    [ calculateMinHeight theme minimumHeight
-                        |> Css.minHeight
-                    ]
-
-        sharedAttributes =
-            [ Events.onInput model.onInput
-            , Attributes.id (generateId model.label)
-            , Attributes.css
+    in
+    Html.styled Html.div
+        [ Css.position Css.relative ]
+        []
+        [ Html.styled (Html.node "nri-textarea-v3")
+            [ Css.display Css.block ]
+            autoresizeAttrs
+            [ Html.styled Html.textarea
                 [ InputStyles.input theme model.isInError
+                , Css.boxSizing Css.borderBox
+                , case model.height of
+                    AutoResize minimumHeight ->
+                        Css.minHeight (calculateMinHeight theme minimumHeight)
+
+                    Fixed ->
+                        Css.batch []
```

Comment:
I think that this actually got introduced in a previous PR, but it looks like a height attribute got deleted here so the textareas are super squashy before they expand!

<img width="743" alt="screen shot 2018-06-05 at 4 12 34 pm" src="https://user-images.githubusercontent.com/13489089/41007574-46394a6c-68db-11e8-8d9c-b81017543751.png">

In the monolith styles, the "Standard" and "ContentCreation" text areas are minheight 100px (see `Nri/Inputs/Styles.elm` line 97), and the "Writing" text area is 150px. Let me know if you wanna fix it here or if I should make a followup PR to fix!

Reply:
Done over here: https://github.com/NoRedInk/noredink-ui/pull/71